src/app/shared/shared/shared.module.ts

```diff
@@ -1,12 +0,0 @@
-import { NgModule } from '@angular/core';
```

Comment:
why this module contains that much changes?

Reply:
Shared module is used to contain shared components, components that are reused in the app like header, footer or social modal component. I suggest reusable components to be imported into the shared.module.ts and then use that module into the lazy loaded modules